src/CTA.Rules.Config/Constants.cs

```diff
@@ -89,5 +89,7 @@ public class Constants
         public const string key = "key";
         public const string value = "value";
         public const string ConnectionStrings = "ConnectionStrings";
+
+        public static readonly List<string> supportedMethodModifiers = new List<string>() { "public", "internal", "protected", "private", "abstract", "extern", "override", "static", "unsafe", "virtual", "async" };
```

Comment:
nit: capitalize static variables/constants

Reply:
Capitalized as suggested, we should make the other variables also match this requirement.